### DISCUSSION TITLE
Bouton reminder > meilleure place

### DISCUSSION THREAD

**Discussion by adriens** [lschaeffer313/domaine-nc-mobile#68] [Polls]
# :grey_question: Contexte

Actuellement, il faut un peu scroller pour lire certains contenus à cause du bouton de reminder cf : 

- https://github.com/lschaeffer313/domaine-nc-mobile/issues/54

![image](https://user-images.githubusercontent.com/5235127/204917924-e18d2a62-88ca-40d4-9899-237d23d01ae9.png)

# :dart: Objectif

Trouver une meilleure solution

**Comment by HakumenNC** [comment]
Petite proposition de mock :

<img src="https://user-images.githubusercontent.com/2644417/205416905-c4601852-90d5-4d76-908d-0926673264ef.png" width=300>

**Reply by adriens** [reply]
C'est super clean et minimaliste, j'aime bcp
